<testsuite name="admin.widget.6W_A02.AW02Create New Widget Step1" tests="3" errors="3" failures="0" skipped="0" time="35.073808" timestamp="2024-05-15T15:33:03.695565" hostname="Kentas-MacBook-Pro.local"><testcase classname="admin.widget.6W_A02.AW02Create New Widget Step1" name="AW02-01ウィジェット名称を設定する Set the widget name" status="failed" time="35.033623"><error type="NoSuchElementException" message="Message: no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;/html/body/div[1]/div/div/div/div/div[2]/div[1]/div/div[2]&quot;}&#10;  (Session info: chrome=124.0.6367.203); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception&#10;Stacktrace:&#10;0   chromedriver                        0x0000000104c62940 chromedriver + 4368704&#10;1   chromedriver                        0x0000000104c5add4 chromedriver + 4337108&#10;2   chromedriver                        0x000000010487ec04 chromedriver + 289796&#10;3   chromedriver                        0x00000001048c0e00 chromedriver + 560640&#10;4   chromedriver                        0x00000001048f95ec chromedriver + 792044&#10;5   chromedriver                        0x00000001048b5ab4 chromedriver + 514740&#10;6   chromedriver                        0x00000001048b650c chromedriver + 517388&#10;7   chromedriver                        0x0000000104c26e5c chromedriver + 4124252&#10;8   chromedriver                        0x0000000104c2bc4c chromedriver + 4144204&#10;9   chromedriver                        0x0000000104c0c824 chromedriver + 4016164&#10;10  chromedriver                        0x0000000104c2c57c chromedriver + 4146556&#10;11  chromedriver                        0x0000000104bfe2d8 chromedriver + 3957464&#10;12  chromedriver                        0x0000000104c4bec4 chromedriver + 4275908&#10;13  chromedriver                        0x0000000104c4c040 chromedriver + 4276288&#10;14  chromedriver                        0x0000000104c5aa34 chromedriver + 4336180&#10;15  libsystem_pthread.dylib             0x000000018e18ef94 _pthread_start + 136&#10;16  libsystem_pthread.dylib             0x000000018e189d34 thread_start + 8">
<![CDATA[
Failing step: When ウィジェット名横のペンシルボタンから名称を変更し、✓ボタンを押す Change the name from the pencil button next to the widget name and press the ✓ button. ... failed in 0.016s
Location: 6W_A02.feature:7
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/behave/model.py", line 1812, in run
    match.run(runner.context)
  File "/opt/homebrew/lib/python3.11/site-packages/behave/matchers.py", line 103, in run
    self.func(context, *args, **kwargs)
  File "../../steps/6W_A02.py", line 49, in chinsara_G
    driver.find_element(By.XPATH,'/html/body/div[1]/div/div/div/div/div[2]/div[1]/div/div[2]').click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/opt/homebrew/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[1]/div/div/div/div/div[2]/div[1]/div/div[2]"}
  (Session info: chrome=124.0.6367.203); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
0   chromedriver                        0x0000000104c62940 chromedriver + 4368704
1   chromedriver                        0x0000000104c5add4 chromedriver + 4337108
2   chromedriver                        0x000000010487ec04 chromedriver + 289796
3   chromedriver                        0x00000001048c0e00 chromedriver + 560640
4   chromedriver                        0x00000001048f95ec chromedriver + 792044
5   chromedriver                        0x00000001048b5ab4 chromedriver + 514740
6   chromedriver                        0x00000001048b650c chromedriver + 517388
7   chromedriver                        0x0000000104c26e5c chromedriver + 4124252
8   chromedriver                        0x0000000104c2bc4c chromedriver + 4144204
9   chromedriver                        0x0000000104c0c824 chromedriver + 4016164
10  chromedriver                        0x0000000104c2c57c chromedriver + 4146556
11  chromedriver                        0x0000000104bfe2d8 chromedriver + 3957464
12  chromedriver                        0x0000000104c4bec4 chromedriver + 4275908
13  chromedriver                        0x0000000104c4c040 chromedriver + 4276288
14  chromedriver                        0x0000000104c5aa34 chromedriver + 4336180
15  libsystem_pthread.dylib             0x000000018e18ef94 _pthread_start + 136
16  libsystem_pthread.dylib             0x000000018e189d34 thread_start + 8

]]>
</error><system-out>
<![CDATA[
@scenario.begin

  @BDDTEST-GPT-332 @BDD_autmation
  Scenario: AW02-01ウィジェット名称を設定する Set the widget name
    Given ウィジェット番号がデフォルトで設定されている Widget number is set by default ... passed in 35.018s
    When ウィジェット名横のペンシルボタンから名称を変更し、✓ボタンを押す Change the name from the pencil button next to the widget name and press the ✓ button. ... failed in 0.016s
    Then ウィジェット番号→指定した名前に変更されるWidget number → changed to the specified name. Name appears in chat header and initial messages. ... skipped in 0.000s

@scenario.end
--------------------------------------------------------------------------------
]]>
</system-out></testcase><testcase classname="admin.widget.6W_A02.AW02Create New Widget Step1" name="AW02-02]担当グループを割り当てる Assign a responsible group" status="failed" time="0.01958"><error type="NoSuchElementException" message="Message: no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;/html/body/div[1]/div/div/div/div/div[2]/div[2]/div/div/div[1]/div[1]/div/div[2]/div/div[1]/div&quot;}&#10;  (Session info: chrome=124.0.6367.203); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception&#10;Stacktrace:&#10;0   chromedriver                        0x0000000104c62940 chromedriver + 4368704&#10;1   chromedriver                        0x0000000104c5add4 chromedriver + 4337108&#10;2   chromedriver                        0x000000010487ec04 chromedriver + 289796&#10;3   chromedriver                        0x00000001048c0e00 chromedriver + 560640&#10;4   chromedriver                        0x00000001048f95ec chromedriver + 792044&#10;5   chromedriver                        0x00000001048b5ab4 chromedriver + 514740&#10;6   chromedriver                        0x00000001048b650c chromedriver + 517388&#10;7   chromedriver                        0x0000000104c26e5c chromedriver + 4124252&#10;8   chromedriver                        0x0000000104c2bc4c chromedriver + 4144204&#10;9   chromedriver                        0x0000000104c0c824 chromedriver + 4016164&#10;10  chromedriver                        0x0000000104c2c57c chromedriver + 4146556&#10;11  chromedriver                        0x0000000104bfe2d8 chromedriver + 3957464&#10;12  chromedriver                        0x0000000104c4bec4 chromedriver + 4275908&#10;13  chromedriver                        0x0000000104c4c040 chromedriver + 4276288&#10;14  chromedriver                        0x0000000104c5aa34 chromedriver + 4336180&#10;15  libsystem_pthread.dylib             0x000000018e18ef94 _pthread_start + 136&#10;16  libsystem_pthread.dylib             0x000000018e189d34 thread_start + 8">
<![CDATA[
Failing step: Given 担当グループ選択覧が空白である The group selection box is blank ... failed in 0.020s
Location: 6W_A02.feature:13
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/behave/model.py", line 1812, in run
    match.run(runner.context)
  File "/opt/homebrew/lib/python3.11/site-packages/behave/matchers.py", line 103, in run
    self.func(context, *args, **kwargs)
  File "../../steps/6W_A02.py", line 76, in chinsara_G
    driver.find_element(By.XPATH,'/html/body/div[1]/div/div/div/div/div[2]/div[2]/div/div/div[1]/div[1]/div/div[2]/div/div[1]/div').click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/opt/homebrew/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[1]/div/div/div/div/div[2]/div[2]/div/div/div[1]/div[1]/div/div[2]/div/div[1]/div"}
  (Session info: chrome=124.0.6367.203); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
0   chromedriver                        0x0000000104c62940 chromedriver + 4368704
1   chromedriver                        0x0000000104c5add4 chromedriver + 4337108
2   chromedriver                        0x000000010487ec04 chromedriver + 289796
3   chromedriver                        0x00000001048c0e00 chromedriver + 560640
4   chromedriver                        0x00000001048f95ec chromedriver + 792044
5   chromedriver                        0x00000001048b5ab4 chromedriver + 514740
6   chromedriver                        0x00000001048b650c chromedriver + 517388
7   chromedriver                        0x0000000104c26e5c chromedriver + 4124252
8   chromedriver                        0x0000000104c2bc4c chromedriver + 4144204
9   chromedriver                        0x0000000104c0c824 chromedriver + 4016164
10  chromedriver                        0x0000000104c2c57c chromedriver + 4146556
11  chromedriver                        0x0000000104bfe2d8 chromedriver + 3957464
12  chromedriver                        0x0000000104c4bec4 chromedriver + 4275908
13  chromedriver                        0x0000000104c4c040 chromedriver + 4276288
14  chromedriver                        0x0000000104c5aa34 chromedriver + 4336180
15  libsystem_pthread.dylib             0x000000018e18ef94 _pthread_start + 136
16  libsystem_pthread.dylib             0x000000018e189d34 thread_start + 8

]]>
</error><system-out>
<![CDATA[
@scenario.begin

  @BDDTEST-GPT-333 @BDD_autmation
  Scenario: AW02-02]担当グループを割り当てる Assign a responsible group
    Given 担当グループ選択覧が空白である The group selection box is blank ... failed in 0.020s
    When 担当グループ選択覧の▼をクリックし、グループ名をクリックする Click ▼ in the group selection box, and click the group name. ... skipped in 0.000s
    Then 担当グループが設定される The group in charge is set ... skipped in 0.000s

@scenario.end
--------------------------------------------------------------------------------
]]>
</system-out></testcase><testcase classname="admin.widget.6W_A02.AW02Create New Widget Step1" name="[AW02-03]担当グループの割り当てを外す Unassign a responsible group" status="failed" time="0.020605"><error type="NoSuchElementException" message="Message: no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;/html/body/div[1]/div/div/div/div/div[2]/div[2]/div/div/div[1]/div[1]/div/div[2]/div/div[1]/div/div[4]/i&quot;}&#10;  (Session info: chrome=124.0.6367.203); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception&#10;Stacktrace:&#10;0   chromedriver                        0x0000000104c62940 chromedriver + 4368704&#10;1   chromedriver                        0x0000000104c5add4 chromedriver + 4337108&#10;2   chromedriver                        0x000000010487ec04 chromedriver + 289796&#10;3   chromedriver                        0x00000001048c0e00 chromedriver + 560640&#10;4   chromedriver                        0x00000001048f95ec chromedriver + 792044&#10;5   chromedriver                        0x00000001048b5ab4 chromedriver + 514740&#10;6   chromedriver                        0x00000001048b650c chromedriver + 517388&#10;7   chromedriver                        0x0000000104c26e5c chromedriver + 4124252&#10;8   chromedriver                        0x0000000104c2bc4c chromedriver + 4144204&#10;9   chromedriver                        0x0000000104c0c824 chromedriver + 4016164&#10;10  chromedriver                        0x0000000104c2c57c chromedriver + 4146556&#10;11  chromedriver                        0x0000000104bfe2d8 chromedriver + 3957464&#10;12  chromedriver                        0x0000000104c4bec4 chromedriver + 4275908&#10;13  chromedriver                        0x0000000104c4c040 chromedriver + 4276288&#10;14  chromedriver                        0x0000000104c5aa34 chromedriver + 4336180&#10;15  libsystem_pthread.dylib             0x000000018e18ef94 _pthread_start + 136&#10;16  libsystem_pthread.dylib             0x000000018e189d34 thread_start + 8">
<![CDATA[
Failing step: Given 担当グループが設定されている The group in charge is set ... failed in 0.021s
Location: 6W_A02.feature:20
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.11/site-packages/behave/model.py", line 1812, in run
    match.run(runner.context)
  File "/opt/homebrew/lib/python3.11/site-packages/behave/matchers.py", line 103, in run
    self.func(context, *args, **kwargs)
  File "../../steps/6W_A02.py", line 96, in chinsara_G
    driver.find_element(By.XPATH,'/html/body/div[1]/div/div/div/div/div[2]/div[2]/div/div/div[1]/div[1]/div/div[2]/div/div[1]/div/div[4]/i').click()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/opt/homebrew/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/div[1]/div/div/div/div/div[2]/div[2]/div/div/div[1]/div[1]/div/div[2]/div/div[1]/div/div[4]/i"}
  (Session info: chrome=124.0.6367.203); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
0   chromedriver                        0x0000000104c62940 chromedriver + 4368704
1   chromedriver                        0x0000000104c5add4 chromedriver + 4337108
2   chromedriver                        0x000000010487ec04 chromedriver + 289796
3   chromedriver                        0x00000001048c0e00 chromedriver + 560640
4   chromedriver                        0x00000001048f95ec chromedriver + 792044
5   chromedriver                        0x00000001048b5ab4 chromedriver + 514740
6   chromedriver                        0x00000001048b650c chromedriver + 517388
7   chromedriver                        0x0000000104c26e5c chromedriver + 4124252
8   chromedriver                        0x0000000104c2bc4c chromedriver + 4144204
9   chromedriver                        0x0000000104c0c824 chromedriver + 4016164
10  chromedriver                        0x0000000104c2c57c chromedriver + 4146556
11  chromedriver                        0x0000000104bfe2d8 chromedriver + 3957464
12  chromedriver                        0x0000000104c4bec4 chromedriver + 4275908
13  chromedriver                        0x0000000104c4c040 chromedriver + 4276288
14  chromedriver                        0x0000000104c5aa34 chromedriver + 4336180
15  libsystem_pthread.dylib             0x000000018e18ef94 _pthread_start + 136
16  libsystem_pthread.dylib             0x000000018e189d34 thread_start + 8

]]>
</error><system-out>
<![CDATA[
@scenario.begin

  @BDDTEST-GPT-334 @BDD_autmation
  Scenario: [AW02-03]担当グループの割り当てを外す Unassign a responsible group
    Given 担当グループが設定されている The group in charge is set ... failed in 0.021s
    When 担当グループ選択覧の✕をクリック Click ✕ in the group selection box ... skipped in 0.000s
    Then 担当グループ選択覧が空欄になる The group selection box is blank ... skipped in 0.000s

@scenario.end
--------------------------------------------------------------------------------
]]>
</system-out></testcase></testsuite>